pkg/downloader/manager.go

```diff
@@ -642,6 +643,11 @@ func (m *Manager) parallelRepoUpdate(repos []*repo.Entry) error {
 //
 // If it finds a URL that is "relative", it will prepend the repoURL.
 func (m *Manager) findChartURL(name, version, repoURL string, repos map[string]*repo.ChartRepository) (url, username, password string, err error) {
+	url, username, password, _, err = m.findChartURLWithTLSVerify(name, version, repoURL, repos)
+	return
+}
+
+func (m *Manager) findChartURLWithTLSVerify(name, version, repoURL string, repos map[string]*repo.ChartRepository) (url, username, password string, insecureskiptlsverify bool, err error) {
```

Comment:
Since this function is private, we do not need to introduce a second function. Just roll this functionality into `findChartURL`.